fftypes/types.go

```diff
@@ -340,3 +340,23 @@ func (s *Schema) ColumnNames() []string {
 	}
 	return names
 }
+
+type FeaturesSchema struct {
```

Comment:
Did Erik not aedd something like this?

Reply:
I did. In this file, at line [441](https://github.com/featureform/featureform-enterprise/blob/main/fftypes/types.go#L441), so I'm surprised this didn't trigger a build failure.

Reply:
I did, but I need to bring a lot of my recent work over from enterprise.